getting-started-publishing/creating-descriptions.md

```diff
@@ -3,19 +3,118 @@ info_en: Page about "Getting started - Creating descriptions"
 info_no: Side om "Komme i gang - beskrive data/lage beskrivelser"
 description: Skal gi overordnet informasjon om hvordan lage beskrivelser
 url: https://data.norge.no/getting-started/describing-data
-note: Vurder å slå sammen med "Publishing descriptions"
 ---
 
-# Lage beskrivelser
+# Beskrive data
 
-## Forarbeid: rammeverk for informasjonsforvaltning og Orden i eget hus
+1. [Du lager beskrivelsen selv i RDF-format](#1-lage-beskrivelserrdf-grafer-selv)
+2. [Du bruker registreringsløsningen til Data.norge.no](#2-registreringsløsning)
 
-## Registreringsløsning
+## Forarbeid: Rammeverk for informasjonsforvaltning og Orden i eget hus
 
-## Skrive beskrivelser
+Uavhengig av hvilke metode du går for bør virksomheten få oversikt over hvilke data den sitter på. Digdir har laget en egen veileder, [Orden i eget hus](https://www.digdir.no/informasjonsforvaltning/veileder-orden-i-eget-hus/2716), for hvordan få til dette.
 
-### Ressurser
+## Spesifikasjoner/standarder
 
-- Lenke til RDF-ressurser og eksempler på beskrivelser
+Data.norge.no baserer seg på flere spesifikasjoner og standarder:
 
-## Verktøy
+- Datasett, datatjenester og datakataloger: [DCAT-AP-NO](https://data.norge.no/specification/dcat-ap-no)
+- Begrep: [SKOS-AP-NO](https://data.norge.no/specification/skos-ap-no-begrep)
+- Informasjonsmodeller: [ModellDCAT-AP-NO](https://data.norge.no/specification/modelldcat-ap-no)
+- Tjenester og hendelser: [CPSV-AP-NO](https://data.norge.no/specification/cpsv-ap-no)
+
+Beskrivelsene som lages må være i et RDF-format og i henhold til spesifikasjonene.
+Registreringsløsningen på Data.norge.no genererer RDF-grafer/beskrivelser i henhold til spesifikasjonen, men løsningen støtter ikke alle feltene som fins i spesifikasjonene.
+
+## 1. Lage beskrivelser/RDF-grafer selv
+
+Bruk disse guidene for å lære mer om hvordan lage datasettbeskrivelser:
+
+- [Veileder for beskrivelse av datasett, datatjenester og datakataloger](https://informasjonsforvaltning.github.io/veileder-beskrivelse-av-datasett/#beskrivelse-av-datasett)
+- [Veileder for orden i eget hus - Steg 5: Beskrive](https://www.digdir.no/informasjonsforvaltning/steg-5-beskrive/2724)
+- [Kursmateriell til workshop i RDF og datasettbeskrivelser](https://github.com/fellesdatakatalog/sikt-workshop/blob/main/ressurshefte/ressurshefte.md)
+
+For de andre type ressursene (begrep, informasjonsmodeller, tjenester og hendelser) fins det eksempler på beskrivelser på [Data.norge.no sitt "Showroom"](https://data.norge.no/showroom/overview). Man kan også støtte seg på den aktuelle spesifikasjonen og eksemplene i den. De fleste av eksemplene er skrevet i RDF-formatet kalt Turtle og har overføringsverdi til de andre typene.
```

Comment:
Burde vi ha som ambisjon å presentere de ulike katalogene for seg, slik at vi kan gi litt tips og triks og relevante lenker som passer til konteksten brukerne er i?

Reply:
Ja, det tror jeg er lurt. Vi har snakket litt om håndbøker for hver ressurstype, men satte det som en nice-to-have på prioriteringslisten for denne gang. Skal se hvor langt vi kommer. Enig i at det er lurt å skille mellom kontekstene, en som er interessert i begrepsarbeid er gjerne mindre opptatt av informasjonsmodeller.